web/test/factories.ts

```diff
@@ -1,3 +1,4 @@
+import { states } from "@/components/tasks/business-formation/StateDropdown";
```

Comment:
To once again make my point about moving this `states` export - it is very strange to see a component being important into a test-generator file. And weird to see .tsx being imported into .ts